src/utils/theme.css

```
@@ -144,6 +139,9 @@ pre[data-line] {
 }
 
 .gatsby-highlight {
+  margin-bottom: 1.75rem;
+  border-radius: 10px;
+  background: #011627;
```

```suggestion
  background: #011627;
  -webkit-overflow-scrolling: touch;
```
Momentum scroll for iOS

---

Sweet! It works like a charm! Thanks @yusufff